docs/project-overview.md

```diff
@@ -0,0 +1,96 @@
+---
+date-generated: 2020-03-30T13:41:16
+tool: NuGetTasks.GenerateMarkdownDoc
+---
+
+
+# NuGet Project Overview
+
+Below is a list of projects contained in the NuGet.Client repo, organized by source and test projects
```

Comment:
Use punctuation at the end.

Reply:
> source and test projects

below you call them product projects...i think that's a better name, so maybe use it here if you agree?

Reply:
Agree. Changed to `...by product and test projects.`.